src/replaceCustomValues.ts

```diff
@@ -181,7 +182,7 @@ export function getElementSubmissionValue({
       )
     }
     case 'abn': {
-      return value?.ABN.identifierValue
```

Comment:
We should use `getABNNumberFromABNRecord` here instead. Since this line was already returning either a string or undefined, we can see the value may be undefined. Should look something like 
```
return value ? getABNNumberFromABNRecord(value) : undefined
```